pages/lists/[listId]/contributors.tsx

```diff
@@ -1,9 +1,110 @@
+import { GetServerSidePropsContext } from "next";
+import { createPagesServerClient } from "@supabase/auth-helpers-nextjs";
 import ListPageLayout from "layouts/lists";
+import ContributorTable from "components/organisms/ContributorsTable/contributors-table";
+import PaginationResults from "components/molecules/PaginationResults/pagination-result";
+import ContributorListTableHeaders from "components/molecules/ContributorListTableHeader/contributor-list-table-header";
+import Pagination from "components/molecules/Pagination/pagination";
+import { fetchApiData, validateListPath } from "helpers/fetchApiData";
+import Error from "components/atoms/Error/Error";
+import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
+import { convertToContributors, useContributorsList } from "lib/hooks/api/useContributorList";
 
-const ListContributors = () => {
-  return <div>List Contributors</div>;
+export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
+  const supabase = createPagesServerClient(ctx);
+
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
+  const bearerToken = session ? session.access_token : "";
+
+  const { listId } = ctx.params as { listId: string };
+  const limit = 10; // Can pull this from the querystring in the future
+  const [{ data, error: contributorListError }, { data: list, error }] = await Promise.all([
+    fetchApiData<PagedData<DBListContributor>>({
+      path: `lists/${listId}/contributors?limit=${limit}`,
+      bearerToken,
+      pathValidator: validateListPath,
+    }),
+    fetchApiData<DBList>({ path: `lists/${listId}`, bearerToken, pathValidator: validateListPath }),
+  ]);
+
+  if (error?.status === 404) {
+    return {
+      notFound: true,
+    };
+  }
```

Comment:
I'm redirecting to the not found page, but do we have something to handle this already?

Reply:
Nothing currently